templates/zerver/app/navbar_alerts.html

```diff
@@ -48,4 +48,23 @@
             {% endtrans %}
         </div>
     </div>
+    <div data-process="bankruptcy" class="alert alert-info brankruptcy">
+        <span class="close" data-dismiss="alert" aria-label="{{ _('Close') }}">&times;</span>
+        <div data-step="1">
+            {% trans count=page_params.unread_msgs.count %}
+            It's been a while! Since you were last here, you received {{ count }} messages.
+            Do you want to skip to your latest messages?
```

Comment:
We should probably make this prompt: "You have {{ count }} unread messages!  Do you want to mark them all as read to start over from a clean slate?"